docs/tutorials/services/constrain-motion.md

```diff
@@ -0,0 +1,408 @@
+
+If you are using a `fake` gripper, there is no real hardware to calibrate and you can [continue this tutorial](#use-a-transform-to-represent-a-drinking-cup).
+
+If you are using a real arm and gripper, use the **Control** tab in the [Viam app](https://app.viam.com/) to move the gripper, look at its reported orientations, and map them to its orientation in the real world.
+If the axes are different from those described above, take these differences into account in your code.
+
+## Use a transform to represent a drinking cup
+
+Imagine your cup is 120 millimeters tall and 90 millimeters in diameter.
+You need to take this space into account to avoid bumping objects on the table with the cup.
+
+You can pass transforms to the [Motion Service `move` method](../../../services/motion/#move) to represent objects that are connected to the robot but are not actual robotic components.
+To represent the drinking cup held in your robot's gripper, add this to your code:
+
+```python {class="line-numbers linkable-line-numbers"}
+# Create a transform to represent the cup as a 120mm tall, 45mm radius capsule shape
+cup_geometry = Geometry(center=Pose(x=0, y=0, z=155),
+    capsule=Capsule(radius_mm=45, length_mm=120))
+transforms = [
+    # Name the reference frame "cup" and point its long axis along the y axis of the gripper
+    Transform(reference_frame="cup", pose_in_observer_frame=PoseInFrame(
+        reference_frame="myGripper",
+        pose=Pose(x=0, y=0, z=45, o_x=0, o_y=-1, o_z=0, theta=0)),
+        physical_object=cup_geometry)
+]
+```
+
+The 155mm offset along the gripper's Z axis accounts for the radius of the gripper (45mm), plus the distance from the gripper's origin (where it meets the arm) and what you can think of as the palm of its hand.
+In this example, that distance is 110mm (see diagram above), but if you have a different gripper, change the offset accordingly.
+
+Now that you have created the table and tissue box obstacles and the cup transform, create a WorldState that includes all of them:
+
+```python {class="line-numbers linkable-line-numbers"}
+# Create a WorldState that includes the table and tissue box obstacles, and the cup transform
+world_state = WorldState(obstacles=[obstacles_in_frame], transforms = transforms)
+```
+
+## Define start and end poses
+
+You need to tell the robot where to pick up the cup, and where to put it down.
+
+While the Motion Service has the mathematical ability to plan complex motion from one position to another, around an obstacle, this is computationally intensive.
```

Comment:
I think your suggestion opens a possible interpretation along the lines of: "The Motion Service is generally computationally intensive, whether it's maneuvering around an obstacle or not." I will suggest a rewording of this paragraph to avoid this as well as your nit :)